src/main/java/br/com/faateodoro/nossacasadocodigo/controller/dto/AutorRequest.java

```diff
@@ -0,0 +1,43 @@
+package br.com.faateodoro.nossacasadocodigo.controller.dto;
+
+import br.com.faateodoro.nossacasadocodigo.modelo.Autor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+public class AutorRequest {
+
+    @NotBlank(message = "O nome é obrigatório")
```

Comment:
Fabiano, eu prefiro que você use internacionalização.

É uma boa prática separar as mensagens do código.

Aí, ficaria assim:

```suggestion
    @NotBlank(message = "{autor.nome.obrigatorio}")
```

E no `ValidationMessages.properties` em `src/main/resources`, você coloca a mensagem:

```properties
autor.nome.obrigatorio = O nome é obrigatório
```